packages/app-store/routing-forms/trpc/utils.ts

```diff
@@ -248,14 +239,65 @@ export async function findTeamMembersMatchingAttributeLogicOfRoute(
 
   const teamMembersMatchingAttributeLogicMap = new Map<number, RaqbLogicResult>();
 
-  const [attributesForTeam, getAttributesForTeamTimeTaken] = await aPf(
-    async () => await getAttributesForTeam({ teamId: teamId })
+  const attrsForTeam = form.teamMembers.reduce(
```

Comment:
Reducer to construct the getAttributesFromTeam from the teamMembersWithAttributeOptions